caom2utils/caom2utils/caom2blueprint.py

```diff
@@ -3167,70 +3592,85 @@ def _has_data_array(self, header):
             return False
         return True
 
-    @staticmethod
-    def _add_keywords(keywords, current, to_set):
-        """
-        Common code for adding keywords to a CAOM2 entity, capturing all
-        the weird metadata cases that happen at CADC.
 
-        :param keywords: Keywords to add to a CAOM2 set.
-        :param current: Existing CAOM2 entity with a keywords attribute.
-        :param to_set: A CAOM2 entity with a keywords attribute.
-        """
-        if keywords:
-            if isinstance(keywords, set):
-                to_set.keywords.update(keywords)
+class HDF5Parser(ContentParser):
+
+    def __init__(
+        self, obs_blueprint, uri, local_f_name, find_roots_here='/sitedata'
+    ):
+        import h5py
```

Comment:
Please add note as to why this import is located here.

Reply:
Added.